app/src/main/kotlin/com/intive/picover/profile/view/ProfileScreen.kt

```diff
@@ -84,9 +65,7 @@ fun ProfileScreen(viewModel: ProfileViewModel) {
 						)
 					},
 					onEditNameClick = {
-						coroutineScope.launch {
-							sheetState.show()
-						}
+						showProfileUpdateBottomSheet = true
```

Comment:
As an improvement we can think about dedicated delegated property to be used in  way:
- val profileUpdateBottomSheet by bottomSheetVisibility() // or "Presence" etc.
- profileUpdateBottomSheet.show()
- profileUpdateBottomSheet.hide()

Reply:
Wouldn't it be the same as using this?
val sheetState = rememberModalBottomSheetState()
val coroutineScope = rememberCoroutineScope()

If we create a class to delegate those booleans with extended ReadWriteProperty I believe we will still need to use rememberSaveable so it would cause a lot of the code and not sure if it is worth the struggle. 

Please see: https://gist.github.com/kmaslowiec/ae22352b61a6fea4a21b79cfe0416819

Reply:
`BottomSheet` visibility and `BottomSheet` state are two different things. If you look at the documentation https://developer.android.com/jetpack/compose/components/bottom-sheets you can see that those properties are used together for different purposes, so I think that we can try to make delegated property for visibility.

Reply:
But it's a separate topic, out of the scope of this PR